kelvinWu/slidingwindow/lc209/lc209.js

```diff
@@ -1,5 +1,26 @@
+// Problem Statement #
+
+// Given an array of positive numbers and a positive number ‘S,’ find the length of the smallest contiguous subarray whose sum is greater than or equal to ‘S’. Return 0 if no such subarray exists.
+
```

Comment:
Please make sure you're writing out a high level summary of the approach and/or detailed comments explaining the approach.